source/utilities.cc

```diff
@@ -1727,9 +1727,9 @@ namespace aspect
         {
           if (coordinate_values_are_equidistant)
             data[i]
-              = std_cxx14::make_unique<Functions::InterpolatedUniformGridData<dim>> (grid_extent,
-                                                                                     table_intervals,
-                                                                                     data_tables[dim+i]);
+              = std_cxx14::make_unique<aspect::InterpolatedUniformGridData<dim>> (grid_extent,
```

Comment:
If you move the function into a namespace `aspect::Functions::`, then you can avoid this change and, once the class is in deal.II, we can just disable it in aspect without having to change this place.